src/mlpack/methods/range_search/range_search_impl.hpp

```diff
@@ -738,15 +738,13 @@ template<typename MetricType,
                   typename TreeStatType,
                   typename TreeMatType> class TreeType>
 template<typename Archive>
-void RangeSearch<MetricType, MatType, TreeType>::Serialize(
+void RangeSearch<MetricType, MatType, TreeType>::serialize(
     Archive& ar,
     const unsigned int /* version */)
 {
-  using data::CreateNVP;
-
   // Serialize preferences for search.
-  ar & CreateNVP(naive, "naive");
-  ar & CreateNVP(singleMode, "singleMode");
+  ar & BOOST_SERIALIZATION_NVP(naive);
+  ar & BOOST_SERIALIZATION_NVP(singleMode);
```

Comment:
It looks like you missed to refactore ``ar & CreateNVP(referenceSet, "referenceSet");`` in line 768, or ``ar & CreateNVP(metric, "metric");``in line 767.

Reply:
@zoq I replaced CreateNVP with BOOST_SERIALIZATION_NVP which I missed in previous commit but still the build is failing. The first error message seems to occur here https://travis-ci.org/mlpack/mlpack/jobs/281876077#L2351.